go/carmen/carmen.go

```diff
@@ -206,6 +212,7 @@ type HistoricBlockContext interface {
 // context before another one can be opened.
 // Methods of this interface are not thread safe.
 type TransactionContext interface {
+	QueryContext
```

Comment:
This looks unrelated ... if so, please remove this line.

Reply:
ok